wiki/Tournaments/TSC/2020/en.md

```diff
@@ -0,0 +1,123 @@
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2020-01-10/2020-02-09 |
+| Live Drawings | *to be announced* |
+| Qualifiers | 2020-02-22/2020-02-23 |
+| Round of 32 | 2020-02-29/2020-03-01 |
+| Round of 16 | 2020-03-07/2020-03-08 |
+| Quarterfinals | 2020-03-14/2020-03-15 |
+| Semifinals | 2020-03-21/2020-03-22 |
+| Finals | 2020-03-28/2020-03-29 |
+| Grand Finals | 2020-04-04 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge |
+
+## Organisation
+
+The Taiko Suiji Cup 2020 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Slave Manager | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) |
+| Map selector | ![][flag_KR] [Konpaku Sariel](https://osu.ppy.sh/users/533502), ![][flag_TW] [-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910) |
+| Referee | ![][flag_RU] [-dragon67](https://osu.ppy.sh/users/4328734), ![][flag_CL] [-Kazu-](https://osu.ppy.sh/users/920861), ![][flag_HK] [Akali393394](https://osu.ppy.sh/users/9686628), ![][flag_BR] [AnonX32](https://osu.ppy.sh/users/2730270), ![][flag_FR] [Apsuko](https://osu.ppy.sh/users/9313135), ![][flag_DE] [Capu](https://osu.ppy.sh/users/2474015), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_AU] [ill onion](https://osu.ppy.sh/users/8306102), ![][flag_US] [JDrago14](https://osu.ppy.sh/users/7690078), ![][flag_FR] [Mimir](https://osu.ppy.sh/users/7382734), ![][flag_FR] [Satsukel](https://osu.ppy.sh/users/9066390), ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958), ![][flag_MY] [Stupid Idiot](https://osu.ppy.sh/users/8355574), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
+| Commentator | ![][flag_AU] [AmateurMonkeyYT](https://osu.ppy.sh/users/8379046), ![][flag_BR] [AnonX32](https://osu.ppy.sh/users/2730270), ![][flag_US] [Bauregaurd](https://osu.ppy.sh/users/2598555), ![][flag_AU] [Beat43210](https://osu.ppy.sh/users/5664171), ![][flag_US] [Binsu](https://osu.ppy.sh/users/6236911), ![][flag_DE] [Drecksackblase](https://osu.ppy.sh/users/6278008), ![][flag_EC] [Gamelan4](https://osu.ppy.sh/users/9856910), ![][flag_DE] [Heam](https://osu.ppy.sh/users/4705120), ![][flag_DE] [Mew](https://osu.ppy.sh/users/2345156), ![][flag_DE] [QEpicAce](https://osu.ppy.sh/users/9489153), ![][flag_NZ] [Sparxe](https://osu.ppy.sh/users/5750235), ![][flag_GB] [Teezel](https://osu.ppy.sh/users/7528639) |
+| Streamer | ![][flag_BR] [AnonX32](https://osu.ppy.sh/users/2730270), ![][flag_US] [ITotallyGetThat](https://osu.ppy.sh/users/8770622), ![][flag_NL] [oliebol](https://osu.ppy.sh/users/2756335), ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958) |
+| Designer | ![][flag_KR] [POCARI SWEAT](https://osu.ppy.sh/users/5082685) |
+| Statistician | ![][flag_PL] [Acrith](https://osu.ppy.sh/users/389880) |
+| Wiki editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
```

Comment:
capitalise both words in position column

Reply:
dont do that